src/main/java/core/basesyntax/Lottery.java

```diff
@@ -1,8 +1,9 @@
 package core.basesyntax;
 
 public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
This solution is okay, but to make it perfect, you can pass dependencies to the constructor.